Firestore/core/src/firebase/firestore/local/local_write_result.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_LOCAL_LOCAL_WRITE_RESULT_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_LOCAL_LOCAL_WRITE_RESULT_H_
+
+#include <utility>
+
+#include "Firestore/core/src/firebase/firestore/model/document_map.h"
+#include "Firestore/core/src/firebase/firestore/model/types.h"
+
+namespace firebase {
+namespace firestore {
+namespace local {
+
+using model::BatchId;
```

Comment:
Nit: don't use `using` declarations in headers (because every file including this header will be forced to use them, whether they want to or not). This is [forbidden](http://google.github.io/styleguide/cppguide.html#Aliases) by the style guide as well. Rather, the header should use the qualified form (`model::BatchId`, etc.).

In short, prefer using `using` declarations in source (`.cc`/`.mm`) files, but don't use them in header files.

Reply:
Done.